go.mod

```diff
@@ -1,11 +1,9 @@
-module github.com/vektra/mockery
+module github.com/simar7/mockery
```

Comment:
This should match the current repo owner, which in this case it would be you @knqyf263 as it's under your repo. But when I had to create a fork I had to replace it with mine so I could build and test. 

Before you merge, make sure to update this to `knqyf263`

Reply:
I see. I'll wait for the update.

Reply:
I updated here https://github.com/knqyf263/mockery/pull/1/commits/ab9901328e313a263144ffe6589bc0e9c249c7bf